Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -1994,6 +1994,13 @@ theorem prod_int_mod (s : Finset α) (n : ℤ) (f : α → ℤ) :
   (Multiset.prod_int_mod _ _).trans <| by rw [Finset.prod, Multiset.map_map]; rfl
 #align finset.prod_int_mod Finset.prod_int_mod
 
+@[to_additive (attr := simp)]
+theorem smul_prod
+    [CommMonoid α] [Monoid β] [MulAction β α] [IsScalarTower β α α] [SMulCommClass β α α]
+    (s : Finset α) (b : β) :
+    ∏ x in s, b • x = b ^ card s • ∏ x in s, x := by
```

Comment:
Thinking again, `smul_prod` doesn't make sense here because we have `smul_sum : r • ∑ x in s, f x = ∑ x in s, r • f x` which is the other way around. So either we should reverse the direction of the lemmas, or restore your original names.